src/components/Experiment/QuickInput/Tabs/TextInput/TextInputTab.js

```diff
@@ -15,7 +15,7 @@ export default function TextInputTab(props) {
             <textarea
                 value={text}
                 className={getElement("input")}
-                onChange={(e) => textChanged(e)}
+                onChange={(e) => textChanged(e, props?.inputIndex)}
```

Comment:
If this is a single input task, do we still have an `inputIndex`?

Reply:
Yeah but it will be undefined so it will not matter. Should i change that?

Reply:
No, I guess it should be fine. I forgot that `undefined` and `null` aren't equivalent, so the code below should work for both single and multiinputs after all.